content/chapters/compute/lab/README.md

```diff
@@ -0,0 +1,201 @@
+# Compute
+
+The main criterion we use to rank CPUs is their _computation power_, i.e. their ability to crunch numbers and do math.
+Numerous benchmarks exist out there and they are publicly displayed on websites such as [CPUBenchmark](https://www.cpubenchmark.net/).
+
+This benchmark measures the performance of the computer's CPU in a variety of scenarios:
```

Comment:
It explains what is being measured as part of a benchmark to get a better idea about what a CPU does adn why there is a need for an OS. All in all, it's less than 15 lines and then moves on introduce the OS. I don't think it's so elaborate, especially since the links aren't menat to be analysed in detail, but merely skimmed over.

Reply:
OK, you can keep it. My concern is that it focuses on benchmarking and results which may be too much for students at this point to grasp.